api/report_server.thrift

```diff
@@ -36,17 +36,17 @@ typedef list<RunData> RunDataList
 
 //-----------------------------------------------------------------------------
 struct ReportData{
-  1: string              checkerId,       // the qualified id of the checker that reported this
-  2: string              bugHash,         // This is unique id of the concrete report.
-  3: string              checkedFile,     // this is a filepath
-  4: string              checkerMsg,      // description of the bug report
-  5: i64                 reportId,        // id of the report in the current run in the db
-  6: bool                suppressed,      // true if the bug is suppressed
-  7: i64                 fileId,          // unique id of the file the report refers to
-  8: shared.BugPathEvent lastBugPosition  // This contains the range and message of the last item in the symbolic
-                                          // execution step list.
-  9: shared.Severity     severity         // checker severity
-  10: optional string    suppressComment  // suppress commment if report is suppressed
+  1: string               checkerId,       // the qualified id of the checker that reported this
+  2: string               bugHash,         // This is unique id of the concrete report.
+  3: string               checkedFile,     // this is a filepath
+  4: string               checkerMsg,      // description of the bug report
+  5: i64                  reportId,        // id of the report in the current run in the db
+  6: i64                  fileId,          // unique id of the file the report refers to
+  7: shared.BugPathEvent  lastBugPosition  // This contains the range and message of the last item in the symbolic
+                                           // execution step list.
+  8: shared.Severity      severity         // checker severity
+  9: shared.ReviewStatus  reviewStatus     // bug review status.
+  10: string              reviewComment    // review commment if review status is changed.
```

Comment:
We show these values at the bug overview:
![cc_review_status](https://user-images.githubusercontent.com/6695818/29061120-c4a62e26-7c1c-11e7-9a4d-fb5d966f4307.png)
If we remove these values from here, we need to query these values for every report id which isn't very good for performance.